cli_test2.py

```diff
@@ -0,0 +1,49 @@
+'''
+Testing file (TY)
+ - I likely will be writing my own test files
+    since my TextEdit crashes sometimes when loading your code... orz
+'''
+
+#==================================================#
+'''Import'''
+
+import toml
+from logic.common import file_utils as fu
+#	from logic.common import level_reader as lr
+#	from logic.common import level_writer as lw
+#	from logic.common import image_exporter as ie
+from logic.test import test_logic as logic
+
+
+#==================================================#
+
+# I put main() inside a function
+#  in order to comment out single lines without affecting indentation
+def main():
+	'''The main procedure for each CLI'''
+	fu.start_procedures()
+
+	# TOML DEMO
+	fu.debug_root("input/root_dir.toml")
+
+	# File I/O DEMO
+	input_path = fu.get_first_valid_root("input/root_dir.toml")
+	input_path += "d30.xml"
+#	print(input_path)
+
+	fu.end_procedures()
+
+main()
+
+#==================================================#
```

Comment:
I vaguely remember you can "collapse" an entire function-block or comment-block in many IDE, but since I prefer the simplicity of my notepad-alike, I am not able to do these collapsing on my end. To keep my code clean, I choose to use these dividers to divide things into sections more explicitly, which also helps when I'm scrolling through large chunks of code to know which sections I'm currently in.

Anyway, I'm considering writing a simple utility program on my end that can do these conversion automatically for all files within folders within folders etc., e.g. replacing "TAB" with 4 space characters and replacing these giant dividers into blank lines. In a sense, it would kinda be similar to my ColorSwapper that I use before/after uploading/downloading tilesheet - You and I get to work in the different working environments that we're comfortable in, all without needing to changing habits.

Reply:
That seems like overkill though.

I'm alright with using TAB instead of 4 spaces and will switch over.

For these dividers, instead of ================== are you alright with using ----------------------------- ?

Reply:
The utility program is more of a handy all-purpose tool that I would probably use for my other mini-projects too, so don't worry much. Not planning on writing it anytime soon though...

Dividers changed.